code/game/objects/items/stacks/medical.dm

```diff
@@ -144,8 +164,28 @@
 			affecting.germ_level = 0
 
 			user.visible_message("<span class='green'>[user] salves the wounds on [H]'s [affecting.name].</span>", \
-							 	 "<span class='green'>You salve the wounds on [H]'s [affecting.name].</span>" )
+								 "<span class='green'>You salves the wounds on [H]'s [affecting.name].</span>" )
+
+			var/rembrute = max(0, heal_brute - affecting.brute_dam)
+			var/remburn = max(0, heal_burn - affecting.burn_dam)
+			var/nrembrute = rembrute

```

Comment:
What do these variables (`nrembrute` and `nremburn`) actually do? It doesn't seem to be referenced usefully anywhere, all they seem to do is be equal to `rembrute` and `remburn`.